build.gradle

```diff
@@ -423,6 +423,13 @@ subprojects {
       ]
     }
 
+    if ( versions.baseScala.startsWith('2.13') ) {
+      scalaCompileOptions.additionalParameters.removeAll( [
+        "-Xlint:by-name-right-associative",
+        "-Xlint:unsound-match"
```

Comment:
We should add a comment explaining why we are doing this.

Reply:
Done.